test/active_record/persistance_test.rb

```diff
@@ -85,6 +88,15 @@ class TestModelB < ActiveRecord::Base
   end
 
   test '#save attempts another request while in transaction' do
+    webmock(:get, '/test_model_cs/schema').to_return(
+      body: {
+        attributes: {
+          id: {type: 'integer', primary_key: true, null: false, array: false},
+          name: {type: 'string', primary_key: false, null: true, array: false}
+        }
+      }.to_json,
+      headers: { 'StandardAPI-Version' => '6.0.0.29' }
+    )
```

Comment:
Shouldn't all the test be updated to this as the default, and this be a test for something < 6.0.0.29 and >= 5.0.0.5

Reply:
If this is now standard and there is a test for 5.0.0.5 this can go

Reply:
I think just keep a test of all version in here no?